src/main/java/com/amazonaws/services/neptune/cluster/AddCloneTask.java

```
@@ -432,11 +476,37 @@ private void createInstance(String name,
             } catch (InterruptedException e) {
                 e.printStackTrace();
             }
-            instanceStatus = neptune.describeDBInstances(DescribeDbInstancesRequest.builder()
-                            .dbInstanceIdentifier(targetDbInstance.dbInstanceIdentifier()).build())
-                    .dbInstances()
-                    .get(0)
-                    .dbInstanceStatus();
+            
+            try {
+                DescribeDbInstancesRequest describeRequest = DescribeDbInstancesRequest.builder()
+                        .dbInstanceIdentifier(targetDbInstance.dbInstanceIdentifier())
+                        .build();
+                        
+                instanceStatus = neptune.describeDBInstances(describeRequest)
+                        .dbInstances()
+                        .get(0)
+                        .dbInstanceStatus();
+                        
+                logger.debug("Instance {} status: {}", targetDbInstance.dbInstanceIdentifier(), instanceStatus);
+            } catch (NeptuneException e) {
```

Was it intentional to change the error handling here to potentially swallow the exception?

---

It was unintended, thanks for catching that. There is a bit of an open question still about if failing to fully populate the cloned cluster should be a fatal error, or if it's best to continue the export with a partial cluster. This logging PR should not attempt to answer that question and should not impact this behaviour though. I have updated to rethrow the error after logging.